src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,94 @@
+package wolox.training.models;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.repositories.BookRepository;
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+public class User {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    private  String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @Column(nullable = false)
+    @ManyToMany(cascade = { CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name = "book_user",
```

Comment:
Just so you know, the JoinTable annotation is not necessary unless you want to specify the names of the columns used and the table name itself, given that with the ManyToMany annotation, a join table is inherently created if nothing is specified :)